docs/en/latest/terminology/plugin.md

```diff
@@ -95,6 +95,43 @@ the configuration above means customizing the error response from the jwt-auth p
 | Name         | Type | Description |
 |--------------|------|-------------|
 | error_response | string/object  | Custom error response |
+| priority       | integer        | Custom plugin priority |
+
+### Custom Plugin Priority
+
+All plugins have a default priority, but it is possible to customize the plugin priority to change the plugin's execution order.
+
+```json
+ {
+    "serverless-post-function": {
+        "_meta": {
```

Comment:
As I already mentioned, this is poor Developer Experience. It makes the use of the configuration more complex for no reason except that it's easier to implement.

Reply:
I also think `priority` should not be in `_meta`, it doesn't fit the definition of `_meta`, it only works for plugin bound objects. How about a custom priority directly as a base property of the plugin? ref: https://github.com/apache/apisix/issues/6580#issuecomment-1089871187

cc @spacewander @membphis

Reply:
The plugin property will affect every plugin configuration, not just what we configure for.

Reply:
After discussing with @spacewander , I changed my mind.

`_meta` is based on the concept of abstraction of all plugin configurations, which are available to all plugins. So to distinguish it, it should have a separate configuration field, i.e. `_meta`.

Reply:
Sorry, but again, you speak from an _engineering_ perspective. I'm talking from a _user_ perspective. As a user, it's not fundamental whether the configuration parameter can be used across multiple plugins.

Here, I need to know **and** remember:

1. The parameter name, _i.e._, `priority`
2. The parent's name, _i.e._, `meta`
3. The prefix, _i.e._, `_` 

Only #1 is mandatory. You make #2 and #3 necessary and impair the ease of use for engineering reasons. While I understand there's a tension between the ease of implementation and ease of use, I don't see any good reason to force those additional requirements onto users at this point.

Reply:
Yes, it is most directly to make `priority` one of the properties of the plugin. just like `disable`, what your option? @spacewander

Reply:
No. Setting the new field directly on the configuration may override people's current configuration. We should avoid break change

Reply:
I have submitted a PR to explain it in the comment: https://github.com/apache/apisix/pull/7290